APIKit/URLEncodedSerialization.swift

```diff
@@ -9,52 +9,51 @@ private func unescape(string: String) -> String {
 }
 
 public class URLEncodedSerialization {
-    public class func objectFromData(data: NSData, encoding: NSStringEncoding, error: NSErrorPointer) -> AnyObject? {
-        var dictionary: [String: AnyObject]?
-        
-        if let string = NSString(data: data, encoding: encoding) as? String {
-            dictionary = [String: AnyObject]()
-            
-            for pair in string.componentsSeparatedByString("&") {
-                let contents = pair.componentsSeparatedByString("=")
-                
-                if contents.count == 2 {
-                    dictionary?[contents[0]] = unescape(contents[1])
-                }
-            }
+    public enum Error: ErrorType {
```

Comment:
It occurs to me that wrapping this error in `APIKitError` may be nice, like `APIKitError.SerializationError(URLEncodedSerialization.Error)`.